src/design/theoretMaxProd.m

```diff
@@ -1,12 +1,13 @@
-function [ExRxns, MaxTheoOut] = theoretMaxProd(model, criterion, inputrxn, normalize, rxns)
+function [ExRxns, MaxTheoOut] = theoretMaxProd(model, inputrxn, criterion, normalize, rxns)

```

Comment:
@jacekwachowiak, is there any reason for changing the order of input arguments here?

Reply:
Looking at the function: 
inputrxn seems to be non optional, so the reorder makes sense. Just make sure, that there are no other references to it in the remaining code of the toolbox.

Reply:
The current version is that the first 3 arguments are non optional but the second - criterion can be empty '' so that the order remains the same.

Reply:
There are no references to this function in the toolbox, so it is best to order the arguments - taking first two as compulsory and moving 'criterion' to optional.